app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class DownvotesController < ApplicationController
+  def create
+    if current_user
+      @review = Review.find(params[:review_id])
+      @vote = @review.votes.find_or_create_by(user: current_user)
```

Comment:
haha I missed that, and would not expect a write operation to appear as the value of a notice. A better alternative:

```ruby
@vote = @review.votes.find_or_initialize_by(user:current)
if @vote.decrement
```